rhea/spack/packages.yaml

```diff
@@ -20,7 +20,6 @@ packages:
 
   effis:
     version: [develop]
-    variants: "python-type=full"
```

Comment:
Shouldn't this still have `+python` (and maybe `+compose')? The idea here was that one can just say 'spack install effis` and would get an EFFIS that actually works for running jobs.

-- Oh, I guess I see, you made those other variants default, so that's why this isn't needed, right?

Reply:
@germasch That's correct.